src/Symfony/Component/Intl/Tests/DateFormatter/Verification/IntlDateFormatterTest.php

```diff
@@ -45,6 +45,8 @@ public function testFormatWithTimezoneFromEnvironmentVariable()
 
     protected function getDateFormatter($locale, $datetype, $timetype, $timezone = null, $calendar = IntlDateFormatter::GREGORIAN, $pattern = null)
     {
+        IntlTestHelper::requireFullIntl($this, '55.1');
```

Comment:
> The intend is to make tests run more frequently instead of being skipped, especially on my Ubuntu LTS that ships with 55.1, and where tests run fine.

Ok. The problem is that these tests are currently very coupled to a specific ICU version, as data between versions changes quite a lot.

What I will try to do is to find examples that didn't change between recent ICU versions and update test cases. Otherwise we'll need to rollback your change and think of another way of running these tests.